run_service.sh

```diff
@@ -187,63 +187,60 @@ get_on_chain_service_state() {
 store=".trader_runner"
 rpc_path="$store/rpc.txt"
 operator_keys_file="$store/operator_keys.json"
+operator_pkey_path="$store/operator_pkey.txt"
 keys_json="keys.json"
 keys_json_path="$store/$keys_json"
+agent_pkey_path="$store/agent_pkey.txt"
```

Comment:
This will make all the participants' trader runner stores corrupted!

Reply:
Can we fix it with a "patch" before line 243 as follows:

if operator_keys_file exists --> create operator_pkey.txt
if keys_json exists --> create agent_pkey.txt

(create using same approach as in create_storage)

Reply:
Yes, sounds like a good idea. This will make it backwards compatible.